bin/bats

```diff
@@ -3,42 +3,70 @@
 set -e
 
 BATS_READLINK='true'
+BATS_REALPATH=''
 if command -v 'greadlink' >/dev/null; then
   BATS_READLINK='greadlink'
 elif command -v 'readlink' >/dev/null; then
   BATS_READLINK='readlink'
 fi
 
+set +e
+$BATS_READLINK -m $0 &>/dev/null;
+[ $? -ne 0 ] && BATS_REALPATH='realpath'
+set -e
+
+resolve_path() {
+  local path=$1
```

Comment:
Although variables are case sensitive (*and* this is `local`), it might make sense to rename this for clarity as PATH is obviously used for other things. WDYT?

Reply:
What would be a better naming, it is the path we are talking about. I'm open for suggestions but I personally think, `local` and the complete lack of case-y-ness should make it clear that is isn't the PATH we are talking about especially in the function called `resolve_path`.